doc/api/repl.md

```diff
@@ -406,6 +406,8 @@ changes:
   * `useGlobal` {boolean} If `true`, specifies that the default evaluation
      function will use the JavaScript `global` as the context as opposed to
      creating a new separate context for the REPL instance. Defaults to `false`.
```

Comment:
I think it should be adequate to say:

The CLI REPL sets this value to `true`. Defaults to `false` when used programmatically.